commands.go

```diff
@@ -27,9 +27,11 @@ type Cmd struct {
 	// The command error handling strategy
 	ErrorHandling flag.ErrorHandling
 
-	init CmdInitializer
-	name string
-	desc string
+	init     CmdInitializer
+	name     string
+	longName string
```

Comment:
once space is used as a separator instead of comma, this field will no longer be needed.